api/src/main.rs

```diff
@@ -1,11 +1,7 @@
 #[macro_use] extern crate rocket;
-
-#[get("/")]
-fn index() -> &'static str {
-    "Hello, world!"
-}
+use rocket::fs::FileServer;
 
 #[launch]
 fn rocket() -> _ {
-    rocket::build().mount("/", routes![index])
+    rocket::build().mount("/", FileServer::from("../../client/build"))
```

Comment:
Could consider having a `PROJECT_ROOT_PATH` and then adding `client/build` to that? But not a big deal, maybe just something in the future if you have more paths you're working with.

Also in the future you'll need to figure out how to combine API URLs from Rocket with the React client files being served. I'm not sure if you'll need additional configuration but probably fine to figure that out in a future commit.

Reply:
Okay, will add in the future commits.